nixos/modules/services/audio/spotifyd.nix

```
@@ -14,6 +18,16 @@ in
       config = mkOption {
         default = "";
         type = types.lines;
+        description = ''
+          (Deprecated) Configuration for Spotifyd. For syntax and directives, see
+          <link xlink:href="https://github.com/Spotifyd/spotifyd#Configuration"/>.
+        '';
+      };
+
+      settings = mkOption {
+        default = {};
+        type = toml.type;
+        example = { public.bitrate = 320; };
```

My bad, `public` should be `global`.

```suggestion
        example = { global.bitrate = 320; };
```